hapi/src/config/i18n.config.js

```diff
@@ -1,17 +1,9 @@
 const i18n = require('i18next')
+const resources = require('../language')
 
 i18n.init({
-  resources: {
-    en: {
-      translation: {
-        app_name: ''
-      }
-    }
-  },
-  fallbackLng: 'en',
-  interpolation: {
-    escapeValue: false
-  }
+  resources,
+  lng: 'en'
 })
```

Comment:
@adriexnet could you take a look at this please to confirm everything is right.

Before:
```
i18n.init({
  resources: {
    en: {
      translation: {
        app_name: ''
      }
    }
  },
  fallbackLng: 'en',
  interpolation: {
    escapeValue: false
  }
})
```

I made the changes according to [this documentation](https://react.i18next.com/guides/the-drawbacks-of-other-i18n-solutions)